jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java

```diff
@@ -53,11 +53,10 @@ public ProcessInstanceSummary getProcessInstance(String serverTemplateId,
             return null;
         }
 
-        ProcessServicesClient queryServicesClient = getClient(serverTemplateId,
-                                                              ProcessServicesClient.class);
+        QueryServicesClient queryServicesClient = getClient(serverTemplateId,
```

Comment:
@nmirasch there are more call in this class that use the _ProcessServicesClient_ service that I'm wondering if should also be replaced by the _QueryServicesClient_. Ex: _getProcess_, _getProcessInstanceLogs_, _getProcessInstanceActiveNodes_, _getProcessUserTasks_.